scripts/functions/requirements/pclinuxos

```diff
@@ -45,20 +57,21 @@ or there was problem checking if libyaml-devel is available / installed."
 # not sure if this one is needed for pclinuxos, copied from centos
 requirements_pclinuxos_update_system_check_epel()
```

Comment:
or we can add `epel2016` - ask people from the tickets that triger that change what would be the best values to use :)